### ISSUE TITLE
Integrate new heat map for other modality tractability

### ISSUE THREAD

**Issue by andrewhercules** [opentargets/platform#883]
Similar to the work identified in #882, can we please also update the React rewrite version of the Platform with an "Other Clinical Modalities" section and a link to the Platform downloads page?

**Comment by andrewhercules** [comment]
Ticket reassigned to "Tractability widget" epic #897

**Comment by andrewhercules** [comment]
In the 20.04 release, we introduced target tractability data for other modalities - see [here](https://github.com/opentargets/platform/issues/882#issuecomment-614091368) and [here](https://github.com/opentargets/platform/issues/882#issuecomment-616387921) for more information.

Can we please build a new heat map showing the other tractability data?

**Comment by forus** [comment]
@andrewhercules 

> 3. For the text with the link to the tractability data file, could we please remove for GENE-SYMBOL so that the text now reads `To view the underlying evidence for the tractability assessment, please download the Open Targets tractability data file`

> 5. Update the logic so that the link to the tractability data file is shown for all genes - including genes with no tractability data (e.g. AC004997.1) as some of these genes may have data that did not meet the threshold for the tractability assessment pipeline but may still be relevant to our users.

Do these points still apply? If yes, is the link the same? https://www.targetvalidation.org/downloads/data

**Comment by forus** [comment]
PR: https://github.com/opentargets/platform-app/pull/145

**Comment by andrewhercules** [comment]
@forus:

To keep it simple for the MVP, you can ignore both points as they were focussed on the Angular implementation.

Perhaps once the MVP is done, I do want to hide the tractability heatmaps when there is no data for a specific modality so that it doesn't look like data is missing - I'll add that to my backlog notes and open a ticket post -`alpha`

**Comment by andrewhercules** [comment]
Ticket closed as PR approved and merged into `alpha`
